### PR TITLE
Romans 12:19: Do not take revenge, my dear friends, but leave room for God’s wrath... - Hugboxed Donksoft Vendors

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request
Adds a new type of vending machine to replace the donksoft vendors at shift start that cannot load riot darts.
Theoretically means we can put one on box now...

# Why is this good for the game?

I don't think it is. I think the whole thing is an admin issue, but I want toys at the function so here we are.

# Testing

Haven't done yet, ill probably do it. 

# Wiki Documentation
Might mention somewhere that roundstart vendors cant load riot darts.

# Changelog

:cl:  
rscadd: New hugboxed donk-soft vendor for roundstart. Cant load riot darts.
/:cl:

### CODE REVIEW

File: code/modules/vending/toys.dm [hunk 1]
```diff
@@ -41,3 +41,31 @@
 /obj/item/vending_refill/donksoft
 	machine_name = "Donksoft Toy Vendor"
 	icon_state = "refill_donksoft"
+
+/obj/machinery/vending/donksofttoyvendor/hugbox
+	desc = "Ages 8 and up approved vendor that dispenses toys. This one feels noticably less fun..."
```
Comment (AMyriad) [inline code/modules/vending/toys.dm]: "An ages 8 and up approved vendor that dispenses toys. This one feels noticeably less fun..."

**Comment by Hislyna** [comment]
Please rename to Hugsoft Vendor

**Comment by 00ze-cyclone** [comment]
Lame, I prefer having real donksoft but you rarely see it than a lesser bland version you get forcefed with every rounds until the little flavor it has loose all meaning and enjoyment.

**Comment by MajManatee** [comment]
> Lame, I prefer having real donksoft but you rarely see it than a lesser bland version you get forcefed with every rounds until the little flavor it has loose all meaning and enjoyment.

ok

**Comment by LazennG** [comment]
stop coding

**Comment by MajManatee** [comment]
Stop smelling

**Comment by MajManatee** [comment]
I'd like to point out, this will only replace the pre-existing ones on asteroid and I think one other map. They are not the illegal tech one, and the normal guns can still load riot darts. These are a special case to keep the shift start vendors.

**Comment by AMyriad** [comment]
thank you for taking my feedback, I would however still do this:
image

**Comment by JamieD1** [comment]
Seems like most people hate this

**Comment by azzzertyy** [comment]
i want none. i want it gone. it is awful.

**Comment by MajManatee** [comment]
> Seems like most people hate this

since when has that meant anything

**Review by ToasterBiome — APPROVED** [review]
looks good

**Pull request merged**